dapr/proto/operator/v1/operator.proto

```diff
@@ -13,15 +13,25 @@ option go_package = "github.com/dapr/dapr/pkg/proto/operator/v1;operator";
 
 service Operator {
   // Sends events to Dapr sidecars upon component changes.
-  rpc ComponentUpdate (google.protobuf.Empty) returns (stream ComponentUpdateEvent) {}
+  rpc ComponentUpdate (ComponentUpdateRequest) returns (stream ComponentUpdateEvent) {}
```

Comment:
Is this backwards compatible?

Reply:
Our Kubernetes upgrade process accounts for this, the Operator is upgraded to a newer version of the proto. This is backward compatible with running instances. New instances get injected the new Dapr version which is compatible with the new Operator proto.